source/Engine.cpp

```diff
@@ -298,7 +299,7 @@ void Engine::Place()
 		{
 			const Personality &person = ship->GetPersonality();
 			bool hasOwnPlanet = ship->GetPlanet();
-			bool launchesWithPlayer = (ship->IsYours() || planet->CanLand(*ship))
+			bool launchesWithPlayer = ((ship->IsYours() && planet->CanLand(*ship)) || planet->CanLand(*ship))
```

Comment:
If the landable nature is always required, then it doesn't matter if the ship is yours or not:
```suggestion
			bool launchesWithPlayer = planet->CanLand(*ship)
```